src/main/java/org/example/BillCalculator.java

```diff
@@ -0,0 +1,16 @@
+package org.example;
+
+public class BillCalculator {
+
+    public double calculate(double billValue, float serviceCharge) {
+        return billValue + serviceCharge;
+    }
+
+    public double calculate(double billValue, float serviceCharge, double discount) {
```

Comment:
Wczytaj się w treść: *Niech zniżka dotyczy wyłącznie wartości rachunku, a nie opłaty serwisowej. oznacza to, że ta zniżka jest raczej procentowa. Innymi słowy jeśli zniżka będzie np. 20 to rachunek musi być o 20% mniejszy. A ty w tym momencie odejmujesz zniżkę od całości de facto.